interface/main/calendar/modules/PostCalendar/pnadmin.php

```diff
@@ -39,21 +40,20 @@ function postcalendar_admin_modifyconfig($msg = '', $showMenu = true)
 
     $output->SetInputMode(_PNH_VERBATIMINPUT);
 
-    $header = "<html><head><title>" . xlt("Calendar") . "</title></head>";
-    $header .= <<<EOF
-	
-	<body bgcolor=
-EOF;
-    $header .= '"' . attr($GLOBALS['style']['BGCOLOR2']) . '">';
+    $header = "<html><head><title>" . xlt("Calendar") . "</title>";
+    $header .= Header::setupHeader('', false)  . '</head><body>';
+
     $output->Text($header);
-    if ($showMenu) {
-        $output->Text(postcalendar_adminmenu());
-    }
 
     if (!empty($msg)) {
-        $output->Text('<center><div style="padding:5px; border:1px solid green; background-color: lightgreen;">');
+        $output->Text(postcalendar_adminmenu("clearCache"));
+        $output -> Text('<div class="alert alert-success ml-5 mr-5" role="alert">');
```

Comment:
This should be `ml-1 mr-1`. Padding isn't measured in pixels in Bootstrap. [Please read the documentation on how to do this correctly](https://getbootstrap.com/docs/4.4/utilities/spacing/).

Also there was a `<center>` tag here that wasn't addressed by adding the `text-center` class to this element.